### DISCUSSION TITLE
讨论重新开放很有必要🎉

### DISCUSSION THREAD

**Discussion by ababcabcd** [WeNeedHome/SummaryOfLoanSuspension#848] [General]
既然只能在墙外讨论，还顾忌什么政治正确，真正的正确就是让不同的声音能够发出来，正所谓理越辨越明，谈论的人多了，舆情也就扩散了，也才能在一定程度上引起上面的重视。

**Comment by ztex** [comment]
起码活出统战价值吧  和国内一样删评论啥的 关评论真的挺low的 而且每个人信仰不一样  别人有时候说出他们的信仰而已 你不认同 可以点踩 或者无视 或者辩论 逃避关掉感觉巨婴行为 强行要求别人的信仰和自己的一致也挺离谱的
